examples/github_search/flutter_github_search/pubspec.yaml

```diff
@@ -9,8 +9,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_bloc: ^3.0.0
-  url_launcher: ^4.0.3
+  flutter_bloc:
```

Comment:
same comment here about why we need to change to path?

Reply:
I was hoping it would allow to help not introducing changes in flutter_bloc that would break this example and also reducing the number of places to changes deps when releasing a new version of flutter_bloc. For example, here it was still at 3.0.0 whereas the latest is 3.2.0.
What do you think?

Reply:
Actually it's not possible after changing packages to specific versions:

> pub get failed (1; So, because flutter_github_search depends on both flutter_bloc from path and common_github_search from path, version solving failed.)

Putting 3.2.0 specifically.